runtime/common/src/lib.rs

```diff
@@ -30,6 +30,13 @@ pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
 pub const HOURS: BlockNumber = MINUTES * 60;
 pub const DAYS: BlockNumber = HOURS * 24;
 
+pub const UNITS: Balance = 1_000_000_000_000;
+pub const QUID: Balance = UNITS / 30;
+pub const CENTS: Balance = QUID / 100;
+pub const GRAND: Balance = QUID * 1_000;
+pub const MILLICENTS: Balance = CENTS / 1_000;
+pub const DOLLARS: Balance = CENTS / 1_000;
```

Comment:
We are actually sticking to standard units (no quids, cents, dollars) for the parachain templates. We used to have them early on but then decided against having biased names.  Would suggest these values be recalculated in places used in terms of `UNITS` or `MILLIUNITS`

Reply:
Addressed in [9f19c32](https://github.com/paritytech/extended-parachain-template/pull/94/commits/9f19c32601f15acb7cff649cffa95da7c54c90a2).

Please take a look. I set the `enter` operation to 1 DOT and the extend operation to 0.5 DOT, but those values might not necessarily be good enough.